realm/realm-jni/src/io_realm_internal_SharedGroup.cpp

```diff
@@ -323,4 +323,25 @@ JNIEXPORT jlongArray JNICALL Java_io_realm_internal_SharedGroup_nativeGetVersion
     env->SetLongArrayRegion(version_data, 0, 2, version_array);
 
     return version_data;
+}
+
+JNIEXPORT jboolean JNICALL Java_io_realm_internal_SharedGroup_nativeWaitForChange
+  (JNIEnv *env, jobject, jlong native_ptr)
+{
+    TR_ENTER_PTR(native_ptr)
+    SharedGroup* sharedGroup = SG(native_ptr);
+    try {
+        sharedGroup->enable_wait_for_change();
```

Comment:
Is this needed if we never disable it? Looking at the core code it seems so, it just seems like a slightly strange pattern.